src/agent/onefuzz/src/libfuzzer.rs

```diff
@@ -441,6 +441,20 @@ impl LibFuzzer {
     }
 }
 
+#[cfg(target_os = "windows")]
+fn artifact_prefix(fault_dir: impl AsRef<Path>) -> OsString {
```

Comment:
suggest just using `&Path`, caller is already doing `.as_ref()`

```suggestion
fn artifact_prefix(fault_dir: &Path) -> OsString {
```